photon-core/src/main/java/org/photonvision/common/dataflow/networktables/NetworkTablesManager.java

```diff
@@ -253,14 +253,22 @@ private void checkHostnameAndCameraNames() {
         String mac = NetworkUtils.getMacAddress();
         if (!mac.equals(currentMacAddress)) {
             logger.debug("MAC address changed! New MAC address is " + mac + ", was " + currentMacAddress);
+            kCoprocTable.getSubTable(currentMacAddress).getEntry("hostname").unpublish();
+            kCoprocTable.getSubTable(currentMacAddress).getEntry("cameraNames").unpublish();
             currentMacAddress = mac;
         }
-        if (mac.isEmpty()) {
+        if (mac.equals("00-00-00-00-00-00")) {
```

Comment:
Should getMacAddress return an empty string? Is a String the best way to represent a mac address? This feels like a magic string to me